### PR TITLE
UIIN-1152 Move (right to left) and confirm items and holdings or items to a new instance.

### PR DESCRIPTION
**Pull request by uladislausamets**
## Purpose
https://issues.folio.org/browse/UIIN-1152

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.
- [ ] I've added appropriate record to the CHANGELOG.md
- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [ ] Duplications on new code is 3% or less
  - [ ] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [ ] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: src/ViewInstance.js [hunk 1]
```diff
@@ -50,10 +50,9 @@ class ViewInstance extends React.Component {
     },
     movableItems: {
       type: 'okapi',
-      path: 'inventory/items',
-      fetch: false,
+      path: 'inventory/items/move',
```
Comment (NikitaSedyx) [inline src/ViewInstance.js:53]: are you sure?

File: src/Instance/MoveHoldingContext/MoveHoldingContext.js [hunk 2]
```diff
@@ -59,7 +63,9 @@ const MoveHoldingContext = ({
     if (!result.destination) return;
```
Comment (NikitaSedyx) [inline src/Instance/MoveHoldingContext/MoveHoldingContext.js:63]: it's not readable, please separate var definitions from operators etc

File: src/Instance/MoveHoldingContext/MoveHoldingContext.js [hunk 3]
```diff
@@ -59,7 +63,9 @@ const MoveHoldingContext = ({
...
     const from = result.source.droppableId;
     const to = result.destination.droppableId;
+    const targetTitle = rightInstance.id === to ? rightInstance.title : leftInstance.title;
     setDragToId(to);
+    setMovingTargetName(targetTitle);
```
Comment (NikitaSedyx) [inline src/Instance/MoveHoldingContext/MoveHoldingContext.js:68]: you don't need this as you already have `to` in state and target name can be found in render

File: translations/ui-inventory/en.json [hunk 4]
```diff
@@ -553,5 +553,5 @@
   "moveItems.instance.dropZone": "Drop holding",
   "moveItems.modal.confirmLabel": "Continue",
   "moveItems.modal.title": "Confirm move",
-  "moveItems.modal.message": "{count, number} {count, plural, one {holding} other {holdings}} will bee moved."
+  "moveItems.modal.message": "{count, number} {count, plural, one {holding} other {holdings}} will bee moved to {targetName}"
```
Comment (aliaksei-chumakou) [inline translations/ui-inventory/en.json:556]: bee is a 🐝

File: CHANGELOG.md [hunk 5]
```diff
@@ -2,6 +2,7 @@
 
 ## 5.0.0 (IN PROGRESS)
 
+* Move (right to left) and confirm items and holdings or items to a new instance. Refs UIIN-1152.
```
Comment (ViktorSoroka) [inline CHANGELOG.md:5]: Should be at the bottom of the list.

**Pull request merged**